docs/src/public/userguide/css/customstyles.css

```diff
@@ -183,7 +183,7 @@ table>caption+thead>tr:first-child>th, table>colgroup+thead>tr:first-child>th, t
 }
 
 table>tbody+tbody {
-  b
+  border-top: 0;
```

Comment:
@lesserwhirls, yep, if the same errors exist in the netcdf-java docs css, then it should be fixed (which I'll do shortly).  Let's definitely look at ways to reduce the # of css files so we can share them across docs.

Reply:
Is is possible to make something like a symlink across repositories?

Reply:
It can get a bit complicated. I think what that would look like would be a git submodule, but they are...tricky. https://github.blog/2016-02-01-working-with-submodules/

I think what we want...maybe...is to have a [gem based unidata jekyll theme](https://jekyllrb.com/docs/themes/#understanding-gem-based-themes).